### PR TITLE
Issue 515

### PR DESCRIPTION
**Pull request by Chrissebah**
I removed the hardcoded user profile from HTML and added a placeholder image to the JS.

## Title
#515 

## Describe your changes: 
Removed hardcoded user profile from index.html
Added placeholder image in userDetailsTemplate.js.

## Type of changes:
HTML and JS

## Checklist before requesting a review

* [x] I created a new branch before starting with this ticket?
* [x] I commented the issue after finish in the Frontend Team Board?
* [x] I moved the card to Quality Assurance after finish?

## Did you run into any issues?
- No, issues encountered.

## Screenshots:
- No screenshots to include.

## Feedback
- Yes, I want feedback.

### CODE REVIEW

**Review by Sovak3441 — APPROVED** [review]
Code linted localy with success. Code looks good.

**Comment by Sovak3441** [comment]
Please resolve the Merge Conflict.
Please make sure the repo and branch are synced before any code changes,

File: src/js/views/admin/showUsers.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-// Author: Erlend Bergeng
-// Team: Elastic Meerkat
+// Author: Christian Grøtan
+// Team: Vettakollen
```
Comment (sulenchy) [inline src/js/views/admin/showUsers.js:2]: Consider removing this

File: src/js/templates/profile/userDetailsTemplate.js [hunk 2]
```diff
@@ -41,15 +41,19 @@ export function userDetailsTemplate(data) {
...
     const { logo, name } = data;
-    element.src = logo;
-    element.alt = name + 'logo';
+    imageUrl = logo;
+    element.alt = name + ' logo';
   } else {
     const { avatar, fullName } = data;
-    element.src = avatar;
-    element.alt = fullName + 'avatar';
+    imageUrl = avatar || 'https://cdn4.iconfinder.com/data/icons/music-ui-solid-24px/24/user_account_profile-2-512.png';
```
Comment (sulenchy) [inline src/js/templates/profile/userDetailsTemplate.js:52]: Is the placeholder coming from the prototype?

**Comment by Sovak3441** [comment]
Please resolve the Merge Conflict.

File: src/js/templates/profile/userDetailsTemplate.js (hunk 2 above)
Comment (Chrissebah) [reply src/js/templates/profile/userDetailsTemplate.js:52]: It's just a random placeholder image. Should it have been the same as the one that was hardcoded?

**Comment by Chrissebah** [comment]
I believe it should be good now? Also changed the function for rendering skills to parent to make it a string and not an array.

**Comment by MHerholdt94** [comment]
> I believe it should be good now? Also changed the function for rendering skills to parent to make it a string and not an array.

Make sure to resolve merge conflict(s) as well :)

**Comment by Chrissebah** [comment]
I think the issue is that a different group has changed the index.html and added new hardcoded data. This makes it so that my index.html, the one that doesnt have a bunch of that hardcoded data, is different. Should i just merge it anyway and then push mine?

**Comment by MHerholdt94** [comment]
@Chrissebah sorry for late answer!

If the code has been updated since you started working on your own branch, you should "git pull" to update your code to the newest and make sure your changes, i.e. the removal of the hardcoded details, still presists.

There are also merge conflict(s) in one file here...

mergeconflict

...you can see if resolving this is enough, as long as your other changes (if there are any) is still there :)

**Comment by Olekrr** [comment]
Closing pull requests from sprint 1 as part of sprint 2 cleanup - contact me if you want it re-opened

**Pull request closed without merging**